src/js/home/LayersPanel.jsx

```diff
@@ -24,10 +24,9 @@ const LayerCheckbox = styled.input`
 `;
 
 const sliderThumb = (layerColor) => css`
-  background: ${layerColor || "var(--teal-1)"};
-  border: 2px solid;
-  border-color: var(--white);
-  border-radius: 50%;
+  background: var(--white);
+  border: 4px solid;
+  border-color: ${layerColor || "var(--teal - 1)"};
```

Comment:
We need to explicitly have `border-radius: 0` so that it works on Firefox as well.